src/com/sindhu/webapplication/ToDoList.java

```diff
@@ -5,12 +5,12 @@
 import java.util.List;
 
 public class ToDoList {
-    public static  List<Long> toDoListIds(){
+    public static List<Long> toDoListIds() {
```

Comment:
who is using this function ? this function gets allllll the items from the database. we should never do that. there should always be a username when dealing with To_Do_Lists table.